### DISCUSSION TITLE
E: Unable to locate package securityonion-all

### DISCUSSION THREAD

**Discussion by Pyrasanth** [Security-Onion-Solutions/securityonion#2273] [Unsupported Versions] [answered]
Hi, 

Following up my other question really, in which I can not get the ISO to install, I thought I would go ahead and try to install Security Onion on Ubuntu so I grabbed a copy Ubuntu 18.04 and installed it.  I am following some instructions from here [https://github.com/security-onion-solutions/security-onion/wiki/InstallingOnUbuntu](url) albeit for Ubuntu 16.  I get this from trying to add the repository:

swanson@swanson-desktop:~$ sudo add-apt-repository -y ppa:securityonion/stable
Hit:1 http://gb.archive.ubuntu.com/ubuntu bionic InRelease
Ign:2 http://ppa.launchpad.net/securityonion/stable/ubuntu bionic InRelease
Err:3 http://ppa.launchpad.net/securityonion/stable/ubuntu bionic Release
  404  Not Found [IP: 91.189.95.83 80]
Reading package lists... Done
E: The repository 'http://ppa.launchpad.net/securityonion/stable/ubuntu bionic Release' does not have a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.

When trying to do the update I get a related error:

swanson@swanson-desktop:~$ sudo apt-get update
Hit:1 http://gb.archive.ubuntu.com/ubuntu bionic InRelease
Ign:2 http://ppa.launchpad.net/securityonion/stable/ubuntu bionic InRelease
Err:3 http://ppa.launchpad.net/securityonion/stable/ubuntu bionic Release    
  404  Not Found [IP: 91.189.95.83 80]
Reading package lists... Done                                                
E: The repository 'http://ppa.launchpad.net/securityonion/stable/ubuntu bionic Release' does not have a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.

Which I then think means it can't find what it needs when running the install:

sudo apt-get -y install securityonion-all syslog-ng-core
Reading package lists... Done
Building dependency tree       
Reading state information... Done
E: Unable to locate package securityonion-all

Can anyone help me with the right commands or where to look to correct this?  Is this easier on CentOS?

**Comment by dougburks** [comment, marked as answer]
https://github.com/security-onion-solutions/security-onion/wiki/InstallingOnUbuntu is the link for the old instructions for the legacy Security Onion 16.04 platform which requires Ubuntu 16.04.

If you want to install the new Security Onion 2 on Ubuntu 18.04, please see:
https://docs.securityonion.net/en/2.3/installation.html#installation-on-ubuntu-or-centos
